src/main/java/frc/robot/commands/GetNoteX.java

```diff
@@ -72,14 +74,14 @@ public GetNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVision note
         m_driveTrain = driveTrain;
         initPaths(s_pathLookup.get(targetNote));
 
-        addCommands(new PrintCommand("GetNoteX-- Starting Auto target note: "+ targetNote));
+        addCommands(new PrintCommand("**** GetNoteX-- Starting Auto target note: "+ targetNote));
 
         if (FieldConstants.isCenterNote(targetNote)) {
             // Use note "monitoring" for center notes only
-            addCommands(m_driveTrain.followPath(() -> getInitialPath())
+            addCommands(new DeferredCommand(() -> m_driveTrain.followPath(getInitialPath()), Set.of(m_driveTrain))
                         .alongWith(new MonitorForNote(noteVision, () -> m_driveTrain.getPose(), m_targetNote, this)));
         } else {
-            addCommands(m_driveTrain.followPath(() -> getInitialPath()));
+            addCommands(new DeferredCommand(() -> m_driveTrain.followPath(getInitialPath()), Set.of(m_driveTrain)));
         }
```

Comment:
Why are we only doing MonitorForNote for CenterNotes?

Reply:
It is just too close for the Stage notes. The algorithm (as written) looks over a distance window which won't work over short distances. Seem low risk of our team mate stealing the notes. When we get field experience, we could add this if we want.